packages/example-react/components/Header.js

```diff
@@ -23,4 +23,4 @@ const rule = () => ({
   }
 })
 
-export default createComponent(rule, Header, ['title'])
```

Comment:
i forgot to update it, it was suppose to work because it was suppose to use my proxy ahah